data/css/meson.build

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Endless Mobile, Inc.
+#
+# We only need to have this file in this subdirectory
+# as outputs to custom_target cannot have path separators
+# their names.
+
+sass = find_program('sassc')
+meson_sassc = find_program('../../bin/meson-sassc')
```

Comment:
I'd put the helper script in the same directory (`data/css/`.)

Reply:
Sure